plugins/discovery-gce/src/main/java/org/elasticsearch/plugin/discovery/gce/GceDiscoveryPlugin.java

```diff
@@ -49,8 +51,12 @@
 
 public class GceDiscoveryPlugin extends Plugin implements DiscoveryPlugin, Closeable {
 
+    /** Determines whether settings those reroutes GCE call should be allowed. */
```

Comment:
Can you add that this is for testing purpose only?

Reply:
fixed in cb5ae5f